cdap-docs/developers-manual/source/building-blocks/schedules.rst

```diff
@@ -8,11 +8,22 @@
 Schedules
 ============================================
 
-**Schedules** are used to execute :ref:`Workflows <workflows>` on a recurring basis. You can add a Schedule (such as a
-`crontab schedule <../../reference-manual/javadocs/co/cask/cdap/api/app/AbstractApplication.html#scheduleWorkflow(java.lang.String,%20java.lang.String,%20java.lang.String)>`__)
-to the Workflow::
+**Schedules** are used to execute :ref:`Workflows <workflows>` on a recurring basis. You can add a Schedule
+to the Workflow using
+`this API <../../reference-manual/javadocs/co/cask/cdap/api/app/AbstractApplication.html#scheduleWorkflow(co.cask.cdap.api.schedule.Schedule,%20java.lang.String)>`__
 
-    scheduleWorkflow("FiveHourSchedule", "0 */5 * * *", "MyWorkflow");
+The `Schedules <../../reference-manual/javadocs/co/cask/cdap/api/schedule/Schedules.html>`__
+class contains static methods to create schedules based on time, or schedules based on data availability.
+
+Time Schedules
+==============
+
+**Time Schedules** will execute based on a
+`crontab schedule <../../reference-manual/javadocs/co/cask/cdap/api/schedule/Schedules.html#createTimeSchedule(java.lang.String,%20java.lang.String,%20java.lang.String)>`__.
+You can add such a Schedule to a Workflow::
+
+    scheduleWorkflow(Schedules.createTimeSchedule("FiveHourSchedule", "Schedule running every 5 hours", "0 */5 * * *"),
```

Comment:
If we did that, and applied the right indentation, this line would become:

```
scheduleWorkflow(Schedules.createTimeSchedule("FiveHourSchedule", 
                                                                                   "Schedule running every 5 hours", 
                                                                                   "0 */5 * * *"),
                     "MyWorkflow");
```

Which is not very pretty either, according to me.